assets/css/legend.scss

```diff
@@ -0,0 +1,71 @@
+.legend {
+  margin-left: 30%;
+  margin-bottom: 10px;
+
+  .legend-scale ul {
+    margin: 0 0 5px;
+    padding: 0;
+    float: left;
+    list-style: none;
+  }
+
+  .legend-scale ul li {
+    font-size: 80%;
+    list-style: none;
+    margin-left: 0;
+    line-height: 18px;
+    margin-bottom: 2px;
+  }
+
+  ul.legend-labels li span {
+    display: block;
+    float: left;
+    height: 16px;
+    width: 30px;
+    margin-right: 5px;
+    margin-left: 0;
+  }
+
+  a {
+    color: #777;
+  }
+
+  .mobilise {
+    background: #67a2d3;
+  }
+
+  .available {
+    background: #76bd79;
+  }
+
+  .unavailable {
+    background: #ff767f;
+  }
+
+  .unavailable-for-mobilization {
+    background-color: #7a7a7a !important;
+  }
```

Comment:
You should rebase on master and use/modify selector for centralized classes from [availability-table.scss](https://github.com/crf-devs/resop/blob/master/assets/css/availability-table.scss)

```
.planning-actions-container,
table.availability-table {
  .unknown {
    background-color: #ff767f;
    color: white;
  }

  .available {
    background-color: #76bd79;
    color: white;
  }

  .booked {
    background-color: #67a2d3;
    color: white;
  }

  .locked {
    background-color: #7a7a7a;
    color: white;
  }
}
```